packages/ethereum/types/src/consensus/light_client_header.rs

```diff
@@ -36,6 +36,12 @@ pub struct LightClientUpdate {
     #[serde_as(as = "DisplayFromStr")]
     #[schemars(with = "String")]
     pub signature_slot: u64,
+    /// Trusted slot to verify the new update against
+    // The client **must** have a consensus state for the provided slot
+    #[serde_as(as = "DisplayFromStr")]
+    #[schemars(with = "String")]
+    #[serde(default)]
```

Comment:
I want trusted slot to be allowed to default to 0, at which point we try updating from latest consensus

Reply:
This has been removed as per the discussion with Serdar